doc/api/axis_api.rst

```diff
@@ -1,12 +1,954 @@
-****
-axis
-****
+===================
+ axis and tick API
+===================
 
-
-:mod:`matplotlib.axis`
-======================
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+   :backlinks: entry
 
 .. automodule:: matplotlib.axis
-   :members:
-   :undoc-members:
-   :show-inheritance:
+
+Inheritance
+===========
+
+.. inheritance-diagram:: Tick Ticker XAxis YAxis XTick YTick
+   :private-bases:
```

Comment:
I think this could be moved to the bottom for all figures.
Opening a bug #7190.